packages/next-swc/crates/core/src/page_config.rs

```diff
@@ -82,7 +82,15 @@ impl Fold for PageConfig {
                     }
 
                     if is_config {
-                        if let Some(expr) = &decl.init {
+                        let init = decl.init.as_ref().map(|expr| {
```

Comment:
I think this is redundant.
We use this pass with `swc` **crate** and the swc **crate** invokes custom passes after stripping typescript types.

And this is a public API, because of the binary size.

Reply:
Thanks for the review! I reverted the changes to next-swc.